rfd/0026/README.md

```diff
@@ -80,8 +81,8 @@ state: draft
       - [Failing for other machine-related endpoints used on shared volumes zones](#failing-for-other-machine-related-endpoints-used-on-shared-volumes-zones)
     - [Changes to VMAPI](#changes-to-vmapi)
       - [New `nfserver` `smartdc_role`](#new-nfserver-smartdc_role)
-      - [New internal `required_nfs_volumes` property on VM objects](#new-internal-required_nfs_volumes-property-on-vm-objects)
-      - [New `mounting_volume` parameter for the `ListVms` endpoint](#new-mounting_volume-parameter-for-the-listvms-endpoint)
+      - [New internal `volumes` property on VM objects](#new-volumes-property-on-vm-objects)
```

Comment:
Ah, it seems it's just the TOC hasn't been updated. I use `doctoc --maxlevel 5 README.md` when in the rfd 26's directory to update it.